packages/preview/light-cv/0.1.0/README.md

```diff
@@ -0,0 +1,29 @@
+![alt text](media/image.png)
+
+# CV Template in Typst
+
+This is my CV template written in Typst. You can find a two example CVs in this repository as PDFs:
+
+- [German CV](cv-de.pdf)
+- [English CV](cv-de.typ)
```

Comment:
These links will not work on Universe. Please change them to absolute links or replace them.